tags/links/whitelist.tag

```diff
@@ -3,5 +3,5 @@ aliases: whitelisting, howtowhitelist
 
 ---
 
-See here for information about creating a whitelist with Floodgate.
-https://github.com/GeyserMC/Geyser/wiki/FAQ#how-do-i-add-players-to-the-whitelist-when-using-floodgate
+See here for information about adding Bedrock players to whitelist.json
```

Comment:
```suggestion
See here for information about adding Bedrock players to the server's whitelist:
```
Is this clearer?

Reply:
I mostly just said whitelist.json because its the only form of whitelist it supports (versus bungee whitelist plugins or something)

Reply:
Fair point.